meeting-notes/CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+# Contributing to the Community Management repository
+
+## Intro
+Welcome to UK TRE Community Management Working Group repository! :tada:
+
+☀️ We're excited that you're here and want to contribute. ☀️
+
+We want to ensure that every user and contributor feels welcome, included and supported to participate in the UK TRE community. 
+We hope that the information provided in this file will make it as easy as possible for you to get involved.
+
+There are two main ways to contribute to this repo - via GitHub issues, and via Google Forms. 
+Contributions primarily take the form of creating issues, adding comments and taking part in discussions.
+Specific updates to content and the website are managed by the [Community Management Working Group](https://uktre.org/groups/current/community_management/). 
+
+Please follow these guidelines to make sure your contributions can be easily integrated into the repo. 
+As you start contributing to the UK TRE community, don't forget that your ideas are more important than perfectly curated issues and forms :heart:
+
+If you have any questions that aren't discussed below, please let us know through one of the many ways to get in touch.
+
+## :sparkles: How to contribute
+
+:star: You are acknowledged for all kinds of contributions
+
+Whatever is your availability, there is a way to contribute.
+
+### :wave: I'm busy, I only have 5 minutes
+
+We use GitHub issues on [this project board](https://github.com/orgs/uk-tre/projects/1) to track our work.
+
+Look through the board to troubleshoot an issue or participate in an ongoing discussion by commenting.
+
+You can also share this repository with someone who might be interested to get involved.
+
+For more details on how to get started contributing to the project board, see ‘Contributing to our project board’ below.
+
+### :timer: I've got 15 minutes - tell me what I should do
+
+You can read our [README](./README.md) file to find  more details on the group, as well as details of our aims on the [Community Management Working Group page](https://uktre.org/groups/current/community_management/).
```

Comment:
```suggestion
You can read our [README](../README.md) file to find  more details on the group, as well as details of our aims on the [Community Management Working Group page](https://uktre.org/groups/current/community_management/).
```

Is this new file intentionally under meeting-notes instead of the top-level?